src/features/projectsV2/ProjectsMap/ProjectPopup/index.tsx

```diff
@@ -19,12 +20,12 @@ const ProjectPopup = ({ project, handlePopupLeave, visitProject }: Props) => {
       className={styles.projectPopup}
     >
       <div
+        className={styles.popupContainer}
         onMouseLeave={handlePopupLeave}
         onClick={() => visitProject(project.properties.slug)}
         onKeyDown={() => visitProject(project.properties.slug)}
       >
-        {/* Dummy content to be replaced with actual ProjectSnippet once ready */}
-        <h3>{project.properties.name}</h3>
+        <ProjectSnippet project={project.properties} showPopup={false} />
```

Comment:
For clarity, I think it may make sense to rename the `showPopup` prop to `showTooltipPopups` wherever applicable. Just to avoid confusion with the map popup.